plugin-build/src/main/kotlin/io/sentry/android/gradle/sourcecontext/BundleSourcesTask.kt

```diff
@@ -41,7 +43,9 @@ abstract class BundleSourcesTask : SentryCliExecTask() {
   @get:InputDirectory
   abstract val sourceDir: DirectoryProperty
 
-  @get:InputFile abstract val bundleIdFile: RegularFileProperty
+  @get:InputFile
+  @get:PathSensitive(PathSensitivity.RELATIVE)
```

Comment:
hm, I'm wondering if this should be `PathSensitivity.NONE` instead, because we care about file contents only (the bundle id), but don't care where the file is located?

Reply:
thanks yes, I can adjust

Reply:
adjusted